app/controllers/gyms_controller.rb

```diff
@@ -13,7 +13,7 @@ def show
 
     @gym_users = GymFacade.get_gym_users(params[:id])
     @gym_friends = FriendshipFacade.get_friends_at_gym(params[:id],
```

Comment:
Unrelated, but one thing we can update for readability is changing the `routes.rb` so that it uses `:yelp_gym_id` instead of `:id` as the named route parameter.  Might be a little lift, but I think it would be an improvement for clarity.

Here is the way in the Rails Guides: https://guides.rubyonrails.org/v5.2/routing.html#overriding-named-route-parameters

Reply:
I like this. This app will get very big so to keep from confusing which id is which this would be a good choice. Good catch Scott!

Reply:
Opened a new issue to track resolution on this one - @gunnarrunner  if you are grabbing updates to the `GymSearch` page, this might be a good refactor along the way, while the book is open on that side of the app.